main_routers/characters_router.py

```
@@ -0,0 +1,1202 @@
+                    return JSONResponse({'error': f'上传成功但无法从响应中提取URL'}, status_code=500)
+                
+                # 确保URL有效
+                if not tmp_url.startswith(('http://', 'https://')):
+                    logger.error(f"无效的URL格式: {tmp_url}")
+                    return JSONResponse({'error': f'无效的URL格式: {tmp_url}'}, status_code=500)
+                    
+                # 测试URL是否可访问
+                test_resp = await client.head(tmp_url, timeout=10)
+                if test_resp.status_code >= 400:
+                    logger.error(f"生成的URL无法访问: {tmp_url}, 状态码: {test_resp.status_code}")
+                    return JSONResponse({'error': f'生成的临时URL无法访问，请重试'}, status_code=500)
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)